zuul-core/src/main/java/com/netflix/zuul/context/Debug.java

```diff
@@ -175,7 +175,7 @@ public static void compareContextState(String filterName, SessionContext context
             String prefix = isInbound ? "RESPONSE_INBOUND" : "RESPONSE_OUTBOUND";
             String arrow = "<";
 
-            Debug.addRequestDebug(context, String.format("%s:: %s STATUS: %d ", prefix, arrow, response.getStatus()));
+            Debug.addRequestDebug(context, String.format("%s:: %s STATUS: %s", prefix, arrow, response.getStatus()));
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This code uses &#39;%s&#39; to format int: &#39;getStatus&#39; expression. This is a potential locale-sensitive handling issue. It might cause errors in the handling and processing of the statement at line: 178. Consider formatting this data with &#39;%d&#39; instead.